webpack.config.js

```diff
@@ -35,6 +35,11 @@ module.exports = {
       'process.env': {
         NODE_ENV: JSON.stringify(process.env.NODE_ENV)
       }
+    }),
+
+    new webpack.optimize.UglifyJsPlugin({
+      minimize: true,
```

Comment:
This is undocumented, but works a bloody treat for stripping the whitespace which gives us a nice file size reduction.